### PR TITLE
Made OpenHours hashable

### PR DESCRIPTION
**Pull request by leechristie**

### CODE REVIEW

File: src/main/java/chneau/openhours/OpenHours.java [hunk 1]
```diff
@@ -106,11 +125,7 @@ static Time simplifyHours(String input) {
                 || sec < 0) {
             throw new IllegalArgumentException("input malformed");
         }
-        var t = new Time();
```
Comment (leechristie) [inline src/main/java/chneau/openhours/OpenHours.java:109]: Removed need for default constructor

File: src/main/java/chneau/openhours/OpenHours.java [hunk 2]
```diff
@@ -9,16 +9,15 @@
...
+import java.util.Objects;
 
 public final class OpenHours implements Whenable {
+    
     public static class Time {
         public int hour;
         public int minute;
         public int second;
 
-        public Time() {
```
Comment (leechristie) [inline src/main/java/chneau/openhours/OpenHours.java:19]: not needed, see below

File: src/main/java/chneau/openhours/OpenHours.java [hunk 3]
```diff
@@ -40,6 +39,26 @@ public boolean equals(Object obj) {
 
     final transient List<LocalDateTime> ldts = new ArrayList<>();
 
+    @Override
```
Comment (leechristie) [inline src/main/java/chneau/openhours/OpenHours.java:42]: hashable now

**Comment by chneau** [comment]
Thanks @leechristie that's pretty cool :)

**Pull request merged**